test/tools/fuzzer_common.cpp

```diff
@@ -0,0 +1,121 @@
+/*
+	This file is part of solidity.
+
+	solidity is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	solidity is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with solidity.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include <test/tools/fuzzer_common.h>
+
+using namespace std;
+using namespace dev;
+using namespace dev::eth;
+
+string contains(string const& _haystack, vector<string> const& _needles)
+{
+    for (string const& needle: _needles)
```

Comment:
It looks like this file uses spaces for indentation.

Reply:
switched to tabs.